docs/reference/snapshot-restore/delete-snapshot.asciidoc

```diff
@@ -0,0 +1,48 @@
+[[delete-snapshots]]
+== Delete a snapshot
+++++
+<titleabbrev>Delete snapshot</titleabbrev>
+++++
+
+////
+[source,console]
+-----------------------------------
+PUT /_snapshot/my_backup
+{
+  "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+
+PUT /_snapshot/my_backup/snapshot_1?wait_for_completion=true
+
+PUT /_snapshot/my_backup/snapshot_2?wait_for_completion=true
+
+PUT /_snapshot/my_backup/snapshot_3?wait_for_completion=true
+-----------------------------------
+// TESTSETUP
+
+////
+
+Use the <<delete-snapshot-api,Delete snapshot API>> to delete a snapshot
+from the repository:
+
+[source,console]
+-----------------------------------
+DELETE /_snapshot/my_backup/snapshot_1
+-----------------------------------
+
+When a snapshot is deleted from a repository, {es} deletes all files that are associated with the deleted
+snapshot and not used by any other snapshots. If the delete snapshot operation is started while the snapshot is being
+created, the snapshot process halts and all files created as part of the snapshotting process are
+removed. Use the <<delete-snapshot-api,Delete snapshot API>> to cancel long running snapshot operations that were
+started by mistake.
+
+To delete multiple snapshots from a repository, separate snapshot names by commas or use wildcards:
+
+[source,console]
+-----------------------------------
+DELETE /_snapshot/my_backup/snapshot_2,snapshot_3
+DELETE /_snapshot/my_backup/snap*
+-----------------------------------
```

Comment:
Wildcards and comma-separated values aren't mutually exclusive. `snapshot*,my_other_snapshot*` is valid. However, you could interpret this section that way.

I'd reword and maybe add an example of two comma-separated wildcard expressions.